### ISSUE TITLE
DeprecationWarning: pkg_resources is deprecated

### ISSUE THREAD

**Issue by jaraco** [cherrypy/cherrypy#2019]
Importing cherrypy emits this deprecationwarning:

```
 @ env PYTHONWARNINGS=default::DeprecationWarning pip-run cherrypy -- -c "import cherrypy" 2>1 | grep cherrypy | grep pkg_resources
/var/folders/f2/2plv6q2n7l932m2x004jlw340000gn/T/pip-run-1xjujsc7/cherrypy/__init__.py:60: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
```

**Comment by thezoggy** [comment]
additionally:
```
.\3.11.8\Lib\site-packages\pkg_resources\__init__.py:2846: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('zc')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
    declare_namespace(pkg)
```

**Comment by jaraco** [comment]
> additionally:
> 
> ```
> .\3.11.8\Lib\site-packages\pkg_resources\__init__.py:2846: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('zc')`.
>   Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
>     declare_namespace(pkg)
> ```

This deprecation warning exists upstream (https://github.com/zopefoundation/meta/issues/194). It might be worth silencing it.

**Comment by jaraco** [comment]
This issue is a duplicate of #1973, the fix which was released in v18.10.0.

**Comment by jaraco** [comment]
Also, it appears as if dropping the use of pkg_resources also caused the deprecation warning not to be emitted for `zc`, so hooray!
